translator/trace/jaeger/jaegerproto_to_protospan_test.go

```diff
@@ -0,0 +1,239 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package jaeger
+
+import (
+	"encoding/binary"
+	"testing"
+	"time"
+
+	commonpb "github.com/census-instrumentation/opencensus-proto/gen-go/agent/common/v1"
+	tracepb "github.com/census-instrumentation/opencensus-proto/gen-go/trace/v1"
+	"github.com/google/go-cmp/cmp"
+	model "github.com/jaegertracing/jaeger/model"
+	tracetranslator "github.com/open-telemetry/opentelemetry-service/translator/trace"
+	"github.com/stretchr/testify/assert"
+	"go.opencensus.io/trace"
+
+	"github.com/open-telemetry/opentelemetry-service/consumer/consumerdata"
+	"github.com/open-telemetry/opentelemetry-service/internal"
+)
+
+func TestOpenCensusToJaeger(t *testing.T) {
+	now := time.Unix(1542158650, 536343000).UTC()
+	d10min := 10 * time.Minute
+	d2sec := 2 * time.Second
+	nowPlus10min := now.Add(d10min)
+	nowPlus10min2sec := now.Add(d10min).Add(d2sec)
+
+	jaeger, err := ProtoBatchToOCProto(grpcFixture(now, d10min, d2sec))
+	assert.NoError(t, err, "should not have failed to convert Jaeger Protobuf to OC Proto")
+
+	oc := expectedTraceData(now, nowPlus10min, nowPlus10min2sec)
+
+	if diff := cmp.Diff(jaeger, oc); diff != "" {
+		t.Errorf("Mismatched responses\n-Got +Want:\n\t%s", diff)
```

Comment:
This is different than the other tests: this one compares the translation via Go structures, while the other tests are loading the fixture from a JSON file. As JSON isn't involved in the gRPC endpoint at all, I'm not sure it would bring much value, so, I reused parts from the receiver test.